### PR TITLE
Fix XDEL

### PR DESCRIPTION
**Pull request by art1415926535**
## What do these changes do?

Allows you to pass multiple IDs to XDEL.

## Are there changes in behavior for the user?

No.

## Related issue number

#737

## Checklist

- [x] I think the code is well written
- [x] Unit tests for the changes exist
- [x] Documentation reflects the changes
- [x] If you provide code modification, please add yourself to `CONTRIBUTORS.txt`
  * The format is `<Name> <Surname>`.
  * Please keep alphabetical order, the file is sorted by names. 
- [x] Add a new news fragment into the [`CHANGES/`](../tree/master/CHANGES) folder
  * name it `<issue_id>.<type>` (e.g. `588.bugfix`)
  * if you don't have an `issue_id` change it to the pr id after creating the PR
  * ensure type is one of the following:
    * `.feature`: Signifying a new feature.
    * `.bugfix`: Signifying a bug fix.
    * `.doc`: Signifying a documentation improvement.
    * `.removal`: Signifying a deprecation or removal of public API.
    * `.misc`: A ticket has been closed, but it is not of interest to users.
  * Make sure to use full sentences with correct case and punctuation, for example:
   `Fix issue with non-ascii contents in doctest text files.`

### CODE REVIEW

**Comment by seandstewart** [comment]
@art1415926535 thanks for your submission!

Maintenance for aioredis is back on 👍 

Would you be willing to sign the CLA? Then I think this PR is good to merge.

File: aioredis/commands/streams.py [hunk 1]
```diff
@@ -209,9 +209,9 @@ def xack(self, stream, group_name, id, *ids):
         """Acknowledge a message for a given consumer group"""
         return self.execute(b"XACK", stream, group_name, id, *ids)
 
-    def xdel(self, stream, id):
+    def xdel(self, stream, id, *ids):
         """Removes the specified entries(IDs) from a stream"""
-        return self.execute(b"XDEL", stream, id)
```
Comment (Andrew-Chen-Wang) [inline aioredis/commands/streams.py:214]: You might want to rebase with upstream to handle linting with black.

File: aioredis/commands/streams.py [hunk 2]
```diff
@@ -209,9 +209,9 @@ def xack(self, stream, group_name, id, *ids):
         """Acknowledge a message for a given consumer group"""
         return self.execute(b"XACK", stream, group_name, id, *ids)
 
-    def xdel(self, stream, id):
+    def xdel(self, stream, id, *ids):
```
Comment (Andrew-Chen-Wang) [inline aioredis/commands/streams.py:212]:
```suggestion
    def xdel(self, stream, *ids):
```

Just being concise

File: aioredis/commands/streams.py [hunk 3]
```diff
@@ -209,9 +209,9 @@ def xack(self, stream, group_name, id, *ids):
...
         """Removes the specified entries(IDs) from a stream"""
-        return self.execute(b"XDEL", stream, id)
+        return self.execute(b'XDEL', stream, id, *ids)
```
Comment (Andrew-Chen-Wang) [inline aioredis/commands/streams.py:214]:
```suggestion
        return self.execute(b"XDEL", stream, *ids)
```

Following the above, just being concise. Plus, black linter only accepts "" in this scenario.

**Review by Andrew-Chen-Wang — CHANGES_REQUESTED** [review]

File: aioredis/commands/streams.py (hunk 2 above)
Comment (seandstewart) [reply aioredis/commands/streams.py:212]: While I do agree with this implementation, the rest of the library is implemented the way that @art1415926535  has done.

We should have a separate effort to move to the above pattern.

**Comment by seandstewart** [comment]
Resolved by a separate PR #708

**Pull request closed without merging**